Tic-Tac-Toe/Rule.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+
+namespace Tic_Tac_Toe
+{
+    public static class Rule
+    {
+        
+        public static Boolean DetermineWin(Board gameBoard, CellValue cellValue)
+        {
+            List<Cell> sameValueCells = gameBoard.Cells.FindAll(x => x.Value.Equals(cellValue));
+            if (CheckRow(sameValueCells))
```

Comment:
You can probably combine these conditionals
```suggestion
            if (CheckRow(sameValueCells) || CheckColumn(sameValueCells) ... ) return true
```